lib/scripts.js

```diff
@@ -148,27 +168,33 @@ module.exports = function (gulp, gutil) {
            * example: angular.module("my.module", ["some.dependencies"])
            */
 
-          moduleFiles[module] = [filePath].concat(moduleFiles[module] || []);
-          moduleDependencies[module] = is().empty(dependencies) ?
-              ['ng'] : dependencies;
+            if (moduleFiles[module] === undefined || moduleFiles[module].dev === undefined) {
+                moduleFiles[module] = { dev : [filePath], min : [filePath] };
+            } else {
+                moduleFiles[module].dev.push(filePath);
+                moduleFiles[module].min.push(filePath);
+            }
+            moduleDependencies[module] = is().empty(dependencies) ? ['ng'] : dependencies;
         } else {
           /*
            * found module look up
            * example: angular.module("my.module")
            */
-          if (moduleFiles[module] === undefined) {
-            moduleFiles[module] = [filePath];
-          } else {
-            if (moduleFiles[module].indexOf(filePath) === -1) {
-              moduleFiles[module].push(filePath);
+            if (moduleFiles[module] === undefined || moduleFiles[module].dev === undefined) {
+                moduleFiles[module] = { dev : [filePath], min : [filePath] };
+            } else {
+                if (moduleFiles[module].dev.indexOf(filePath) === -1) {
+                    moduleFiles[module].dev.push(filePath);
+                    moduleFiles[module].min.push(filePath);
+                }
             }
-          }
         }
       }
     }
 
     return gulp.src(['**/*.js', '!lib/**/*.js'], {cwd: src})
       .pipe(isWatch ? plumber()() : gutil.noop())
+      // Use Uglify to get the files in required format to be used in downstream function(createDepsMetadata)
```

Comment:
Can you be more specific here, what is the format that we need the file in for it to work downstream?